### PR TITLE
fix(web): keyboard documentation patch-up

### LINKED ISSUE #4436: The mobile keyboard layouts shown on the help page are cropped off
The mobile keyboard layouts shown on the help page are cropped off when viewing them on mobile devices.

Vertical orientation:
image

Horizontal orientation:
image

While the user can swipe to view the whole keyboard layout, it would be better to ensure that the layouts are completely visible by default.

References:
1. [Original report](https://community.software.sil.org/t/tawallamat-help-images-for-mobile-not-formatted/4396/3)
2. [Sample page with issue](https://help.keyman.com/keyboard/tawallammat_latin/1.2/tawallammat_latin)

### PR DESCRIPTION
**Pull request by jahorton**
Fixes #4436.

Also fixes a minor, 14.0-only issue caused by some of the Web-core work that would have broken things on the beta -> stable transition.

Before:
Screen Shot 2021-02-24 at 12 14 45 PM

After:
Screen Shot 2021-02-24 at 12 07 51 PM

The test page above was introduced by #2802, the previous PR to patch up documentation keyboards.

If viewing the static keyboards on desktop, even mobile-oriented layouts will now dynamically rescale (their width) when the window is resized.  To facilitate this, static keyboard layouts are now always styled with percentage values, never exact pixels.

Since this is for a 'static' keyboard, font scaling may take additional work to get perfect.  Of note, though:  in the above images, font-scaling is set to a standard `1em`.  (Will not dynamically rescale with window resizes.)  Simply adding CSS styling to the root element returned from `BuildDocumentationKeyboard` to scale things up for the static elements is perfectly viable if desired.

### CODE REVIEW

File: web/source/osk/visualKeyboard.ts [hunk 1]
```diff
@@ -2336,7 +2340,9 @@ namespace com.keyman.osk {
 
         for(Ln=0; Ln<keyboardsList.length; Ln++) {
           if(p == keyboardsList[Ln]['KI'].toLowerCase().replace('keyboard_','')) {
-            PKbd=keyboardsList[Ln]; break;
+            // Requires the Keyboard wrapping object now.
+            PKbd = new com.keyman.keyboards.Keyboard(keyboardsList[Ln]);
+            break;
```
Comment (jahorton) [inline web/source/osk/visualKeyboard.ts:2345]: Fixes that minor 14.0-only breakage I mentioned.

Comment (mcdurdin) [reply web/source/osk/visualKeyboard.ts:2343]: What was breaking? Can you describe the symptoms?

Comment (jahorton) [reply web/source/osk/visualKeyboard.ts:2343]: Uh, it just wouldn't display due to a `null` reference.  The value now lives as a field of the `Keyboard` wrapper-class now.

Comment (jahorton) [reply web/source/osk/visualKeyboard.ts:2343]: I obviously forgot to test the documentation-keyboard part when I made the change as part of the Web-core work.

**Comment by darcywong00** [comment]
alas, the Web test build failed

**Comment by jahorton** [comment]
> alas, the Web test build failed

It's been a long time since the last case, but it seems to have been a classic case of BrowserStack instability.  A re-run passed without issue.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.247-beta](https://keyman.com/downloads/releases/beta/14.0.247)

**Pull request merged**